sdk/textanalytics/azure-ai-textanalytics/swagger/README.md

```diff
@@ -0,0 +1,90 @@
+# Text Analytics for Python
+
+To generate this file, simply type
+
+```
+autorest samples/README.md
+```
+
+We automatically hardcode in that this is `python` and `multiapi`.
+
+## Basic
+
+```yaml
+license-header: MICROSOFT_MIT_NO_VERSION
+add-credential: true
+payload-flattening-threshold: 2
+package-name: azure-ai-textanalytics
+clear-output-folder: true
+credential-scopes: https://cognitiveservices.azure.com/.default
+no-namespace-folders: true
+python: true
+multiapi: true
+```
+
+## Multiapi Batch Execution
+
+```yaml $(multiapi)
+batch:
+ - tag: release_3_0
+ - tag: release_3_1_preview.4
+ - multiapiscript: true
+```
+
+## Multiapiscript
+
+``` yaml $(multiapiscript)
+output-folder: ../azure/ai/textanalytics/_generated/
+default-api: v3_0
+clear-output-folder: false
+perform-load: false
+```
+
+## Release 3.0
+
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/TextAnalytics.json
+namespace: azure.ai.textanalytics.v3_0
+output-folder: ../azure/ai/textanalytics/_generated/v3_0
+```
+
+## Release 3.1-Preview.4
+
+These settings apply only when `--tag=release_3_1_preview.4` is specified on the command line.
+
+```yaml $(tag) == 'release_3_1_preview.4'
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json
+namespace: azure.ai.textanalytics.v3_1_preview_4
+output-folder: ../azure/ai/textanalytics/_generated/v3_1_preview_4
+```
+
+### Override Analyze's pager and poller
+
+``` yaml
+directive:
+    -   from: swagger-document
+        where: '$.paths["/analyze"].post'
+        transform: >
+            $["responses"]["200"] = {"description": "dummy schema", "schema": {"$ref": "#/definitions/AnalyzeJobState"}};
+            $["x-python-custom-poller-sync"] = "...._lro.AnalyzeBatchActionsLROPoller";
+            $["x-python-custom-poller-async"] = "....._async_lro.AsyncAnalyzeBatchActionsLROPoller";
+            $["x-python-custom-default-polling-method-sync"] = "...._lro.AnalyzeBatchActionsLROPollingMethod";
+            $["x-python-custom-default-polling-method-async"] = "....._async_lro.AsyncAnalyzeBatchActionsLROPollingMethod";
```

Comment:
is this what we talked about that if we add in the main swagger, all languages will get for free without the need to do customization?

Reply:
not quite. So for python we generate with poller `LROPoller`. Healthcare and analyze need their own custom poller, so I'm generating with their poller, i.e. `AnalyzeBatchActionsLROPoller` instead of `LROPoller`. This is quite python specific